contrib/valgrind.supp

```diff
@@ -149,3 +149,22 @@
    ...
    fun:ib_cm_open_device
 }
+{
+   <insert_a_suppression_name_here>
+   Memcheck:Leak
+   match-leak-kinds: possible
+   fun:calloc
+   obj:/usr/lib64/libibverbs/libhfi1verbs-rdmav22.so
+   obj:/usr/lib64/libibverbs.so.1.5.22.1
```

Comment:
here is exact library version... is it ok?

Reply:
it's not OK !